compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -2956,7 +2956,8 @@ declare_lint_pass! {
         UNSUPPORTED_NAKED_FUNCTIONS,
         MISSING_ABI,
         SEMICOLON_IN_EXPRESSIONS_FROM_MACROS,
-        DISJOINT_CAPTURE_DROP_REORDER,
```

Comment:
we *ideally* don't want to rename it, people are already using this name and it would break their code